programs/exchange/src/math.rs

```diff
@@ -223,10 +233,10 @@ pub fn calculate_max_burned_in_token(asset: &Asset, user_debt: &u64) -> u64 {
         return burned_amount_token.try_into().unwrap();
     } else {
         let burned_amount_token = div_up(
-            (*user_debt as u128)
+            (user_debt as u128)
                 .checked_mul(10u128.pow(PRICE_OFFSET.into()))
                 .unwrap()
-                .checked_div(10u128.pow(decimal_difference.try_into().unwrap()))
+                .checked_div(10u128.pow((-decimal_difference).try_into().unwrap()))
```

Comment:
Why is there minus sign ?

Reply:
if `decimal_difference < 0` is required to replace the expression
`user_debt * 10^(decimal_difference) -> user_debt / 10^(-decimal_difference)`
to avoid floating point division. Similar example is in `calculate_swap_out_amount`